src/components/uiComponents/DashboardMobile.tsx

```diff
@@ -0,0 +1,72 @@
+import LineChart from '../../components/uiComponents/BalanceChart';
+import BarChart from '../../components/uiComponents/BarChart';
+import Card from './Card';
+import NewCard from './NewCard';
+import PieChart from '../../components/uiComponents/PieChart';
+import QuickTransfer from '../../components/uiComponents/QuickTransfer';
+import Transaction from '../../components/uiComponents/Transaction';
+
+const DashboardMobile = () => {
```

Comment:
no component based on breakpoint. We need responsive design in total. Please understand requirements